benchmark/protocol/src/main/scala/Protocol.scala

```diff
@@ -59,7 +59,7 @@ object protocol {
       .codec[MediumW]
       .codec[SmallW]
       .codec[BigW]
-      .specify1[LargeW, LargeW]("identityLarge")
-      .specify1[MediumW, MediumW]("identityMedium")
-      .specify1[BigW, BigW]("identityBig")
+      .specify1("identityLarge", Field.strict[LargeW]("in"), Type[LargeW])
```

Comment:
Really not sure what i feel about this syntax. It doesnt look particular great. There are some other options, but im not sure what i prefer most:

``` scala

// 8/10 gross
.specify1("foo", Field.strict[LargeW]("bar"), Type[LargeW])
// 5/10 gross
.specify1[LargeW]("foo", strict("bar"))
// 4/10 gross, but implicits: blagh.
.specify1[LargeW]("foo", "bar".strict)
// not sure how to infer the result type here?
.specify1("foo", "bar".strict[LargeW])
```